scripts/globals/abilities/despoil.lua

```diff
@@ -20,68 +20,68 @@ local despoilDebuffs =
 }
 
 function onAbilityCheck(player, target, ability)
-    if (player:getFreeSlotsCount() == 0) then
-        return dsp.msg.basic.FULL_INVENTORY, 0;
+    if not player:getFreeSlotsCount() then
```

Comment:
0 is true in lua - unless you changed getFreeSlotsCount to return nil instead of 0, this is not gonna work